spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/procedures/AddFilesProcedure.java

```diff
@@ -172,11 +173,7 @@ private void importFileTable(Table table, Path tableLocation, String format, Map
     } else {
       Preconditions.checkArgument(!partitions.isEmpty(),
           "Cannot find any partitions in table %s", partitions);
```

Comment:
Nit / corner case:

Should we update the precondition message to indicate that it's possible that the filter didn't match any partitions? The current error message now might be kind of confusing to users if the file based table is partitioned. Maybe just `Cannot find any matching partitions in table %s`?

Reply:
Fixed. Thanks!